src/systems/breadcrumbs/Breadcrumbs.cc

```diff
@@ -45,6 +45,16 @@ void Breadcrumbs::Configure(const Entity &_entity,
     EntityComponentManager &_ecm,
     EventManager &_eventMgr)
 {
+  this->maxDeployments =
+      _sdf->Get<int>("max_deployments", this->maxDeployments).first;
+
+  // Exit early if breadcrumbs deployments are not possible.
+  if (this->maxDeployments <= 0)
+  {
+    ignmsg << "Breadcrumbs max deployment is <= 0. Breadcrumbs are disabled."
+      << std::endl;
+  }
```

Comment:
Should there be a return here?
```suggestion
    return;
  }
```

Reply:
Added: 154fdfdd7a46ef4d5a05f37ee0a99725616aa37d